Source/controls/devices/game_controller.cpp

```diff
@@ -238,23 +238,32 @@ GamepadLayout GameController::getLayout(const SDL_Event &event)
 	const int index = event.cdevice.which;
 	const SDL_GameControllerType gamepadType = SDL_GameControllerTypeForIndex(index);
 	switch (gamepadType) {
-#if SDL_VERSION_ATLEAST(2, 0, 16)
-	case SDL_CONTROLLER_TYPE_GOOGLE_STADIA:
-	case SDL_CONTROLLER_TYPE_AMAZON_LUNA:;
+	case SDL_CONTROLLER_TYPE_NINTENDO_SWITCH_PRO:
+#if SDL_VERSION_ATLEAST(2, 0, 23)
+	case SDL_CONTROLLER_TYPE_NINTENDO_SWITCH_JOYCON_LEFT:
+	case SDL_CONTROLLER_TYPE_NINTENDO_SWITCH_JOYCON_RIGH:

```

Comment:
Is it a good idea to support individual joycons? We lose two trigger buttons and a start/select button with just one joycon. Is this adequate for playing Diablo?

Reply:
I'm not entirely sure how it works, my thinking was that it would still behave as the full gamepad. But maybe that is incorrect? We sort of would need someone to test this out. Feel free to ask people on Discord etc.